### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by Coleyra**
Add french translation

### CODE REVIEW

File: src/assets/locales/fr-FR/translation.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
+{
+  "main": {
+    "overall": "Total",
+    "crewmate": "Equipage",
```
Comment (MissingNoShiny) [inline src/assets/locales/fr-FR/translation.json:4]:
```suggestion
    "crewmate": "Équipage",
```

File: src/assets/locales/fr-FR/translation.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    "losses": "Défaites",
+    "crewmate": "Equipage",
+    "impostor": "Imposteur",
+    "resetScores": "Réinitialiser les Scores",
+    "resetRound": "Réinitialiser les Round",
+    "resetAll": "Tout Réinitialiser",
+    "notes": "Notes",
+    "resetNotes": "Réinitialiser les Notes",
+    "lockPlayers": "Vérouiller les Joueurs",
+    "unlockPlayers": "Dévérouiller les Joueurs"
```
Comment (MissingNoShiny) [inline src/assets/locales/fr-FR/translation.json:41]:
```suggestion
    "crewmate": "Équipage",
    "impostor": "Imposteur",
    "resetScores": "Réinitialiser les scores",
    "resetRound": "Réinitialiser la partie",
    "resetAll": "Tout réinitialiser",
    "notes": "Notes",
    "resetNotes": "Réinitialiser les notes",
    "lockPlayers": "Verrouiller les joueurs",
    "unlockPlayers": "Déverrouiller les joueurs"
```

File: src/assets/locales/fr-FR/translation.json [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    "wiringTasks": "Tâches avec câbles",
+    "ventConnections": "Liens de Ventilations",
+    "securityCameras": "Caméras de Sécurité",
+    "cameraZones": "Vision des Caméras",
+    "reactor": "Reacteur",
+    "upperEngine": "Moteur Haut",
+    "lowerEngine": "Moteur Bas",
+    "security": "Sécurité",
+    "medBay": "Infirmerie",
+    "electrical": "Electrique",
```
Comment (MissingNoShiny) [inline src/assets/locales/fr-FR/translation.json:56]:
```suggestion
    "electrical": "Électrique",
```

File: src/assets/locales/fr-FR/translation.json [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    "storage": "Stockage",
+    "cafeteria": "Cafeteria",
```
Comment (MissingNoShiny) [inline src/assets/locales/fr-FR/translation.json:58]:
```suggestion
    "cafeteria": "Cafétéria",
```

File: src/assets/locales/fr-FR/translation.json [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    "admin": "Admin",
+    "communications": "Communications",
+    "shields": "Boucliers",
+    "o2": "O2",
+    "weapons": "Armurerie",
+    "navigation": "Navigation",
+    "launchpad": "Rampe de Lancement",
```
Comment (MissingNoShiny) [inline src/assets/locales/fr-FR/translation.json:65]:
```suggestion
    "launchpad": "Rampe de lancement",
```

File: src/assets/locales/fr-FR/translation.json [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+  },
+  "menu": {
+    "players": "Joueurs",
+    "notes": "Notes",
+    "scores": "Scores",
+    "maps": "Cartes",
+    "menu": "Menu",
+    "settings": "Réglages",
+    "feedback": "Retours",
+    "about": "A propos",
```
Comment (MissingNoShiny) [inline src/assets/locales/fr-FR/translation.json:99]:
```suggestion
    "about": "À propos",
```

File: src/assets/locales/fr-FR/translation.json [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+    "admin": "Admin",
+    "communications": "Communications",
+    "shields": "Boucliers",
+    "o2": "O2",
+    "weapons": "Armurerie",
+    "navigation": "Navigation",
+    "launchpad": "Rampe de Lancement",
+    "laboratory": "Laboratoire",
+    "lockerRoom": "Vestiaire",
+    "office": "Office",
```
Comment (MissingNoShiny) [inline src/assets/locales/fr-FR/translation.json:68]:
```suggestion
    "office": "Bureau",
```

File: src/assets/locales/fr-FR/translation.json [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+    "northeast": "3. Nord-Est",
+    "south": "4. Sud",
+    "southwest": "5. Sud-Ouest",
+    "northwest": "6. Nord-Ouest",
+    "dragInstructions": "Faites glisser les joueurs sur la carte pour les traquer",
+    "resetPlayers": "Réinitialiser les Positions"
+  },
+  "settings": {
+    "title": "Réglages",
+    "uiSettings": "Réglages de l'Interface",
```
Comment (MissingNoShiny) [inline src/assets/locales/fr-FR/translation.json:83]:
```suggestion
    "uiSettings": "Réglages de l'interface",
```

File: src/assets/locales/fr-FR/translation.json [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+    "balcony": "Balcon",
+    "specimenRoom": "Salle d'échantillons",
+    "east": "1. Est",
+    "central": "2. Centre",
+    "northeast": "3. Nord-Est",
+    "south": "4. Sud",
+    "southwest": "5. Sud-Ouest",
+    "northwest": "6. Nord-Ouest",
+    "dragInstructions": "Faites glisser les joueurs sur la carte pour les traquer",
+    "resetPlayers": "Réinitialiser les Positions"
```
Comment (MissingNoShiny) [inline src/assets/locales/fr-FR/translation.json:79]:
```suggestion
    "resetPlayers": "Réinitialiser les positions"
```

File: src/assets/locales/fr-FR/translation.json [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+      "evil": "Méchant",
+      "hitList": "Liste noire",
+      "dead": "Mort",
+      "unknown": "Pas d'information"
+    },
+    "brown": "Marron",
+    "red": "Rouge",
+    "orange": "Orange",
+    "yellow": "Jaune",
+    "lime": "Citron",
```
Comment (MissingNoShiny) [inline src/assets/locales/fr-FR/translation.json:21]: "Citron" by itself is a shade of yellow. I suggest either "Vert citron", "Vert clair", or if both of those don't fit, "Tilleul" (arguably not as good, but shorter).
```suggestion
    "lime": "Vert clair",
```

File: src/assets/locales/fr-FR/translation.json [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+    "greenhouse": "Serre",
+    "balcony": "Balcon",
+    "specimenRoom": "Salle d'échantillons",
+    "east": "1. Est",
+    "central": "2. Centre",
+    "northeast": "3. Nord-Est",
+    "south": "4. Sud",
+    "southwest": "5. Sud-Ouest",
+    "northwest": "6. Nord-Ouest",
+    "dragInstructions": "Faites glisser les joueurs sur la carte pour les traquer",
```
Comment (MissingNoShiny) [inline src/assets/locales/fr-FR/translation.json:78]:
```suggestion
    "dragInstructions": "Faites glisser les joueurs sur la carte pour suivre leur position",
```

**Review by MissingNoShiny — CHANGES_REQUESTED** [review]
Good translation overall. I fixed a few mistakes, missing accents and unnecessary capitalizations.

**Review by Kedyn — APPROVED** [review]
Thank you for your contribution. Please keep in mind that we are working towards few new things and we might be adding/removing some things, so I ask if you would like to keep this translation up to date keep an eye as to what has been changed. I will be trying to open an issue mentioning you and all those who have helped with translations once we are ready to deploy our next version.

PS: Would you mind adding yourselves to the readme, thank you.

**Comment by MissingNoShiny** [comment]
I wouldn't want to steal credit for this translation, as I barely changed anything of importance (and besides, I'm pretty new to contributing to repos that aren't mine, and I believe I don't have write access to this branch?). However, feel free to mention me if you need any further help!

**Pull request merged**